include/Contact.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
-require_once('include/Probe.php');
```

Comment:
I'm more and more pessimistic about us being able to move to a fully autoloaded codebase, so I figured I'd start improving where I can. Unnecessary cascading inclusion eats up to .4s for each request. For a full page it's not a big deal but for pings and updates it's a killer.